src/api/VotingIrregularities.Api/Controllers/Authorization.cs

```diff
@@ -126,7 +126,7 @@ public async Task<IActionResult> NgoAdmin([FromBody] NgoAdminApplicationUser ngo
 
                 if (identity == null) {
                     _logger.LogInformation($"Invalid Phone ({request.User}) or password ({request.Password})");
-                    return BadRequest("{ \"error\": \"A aparut o eroare la logarea in aplicatie. Va rugam sa verificati ca ati introdus corect numarul de telefon si codul de acces, iar daca eroarea persista va rugam contactati serviciul de suport la numarul 0757652712.\" }");
+                    return BadRequest("{ \"error\": \"A aparut o eroare la logarea in aplicatie. Va rugam sa verificati ca ati introdus corect numarul de telefon si codul de acces, iar daca eroarea persista va rugam contactati serviciul de suport la numarul 0767198682.\" }");
```

Comment:
you could return _mobileSecurityOptions.InvalidCredentialsErrorMessage and set it from appsettings with
`{ 'error': 'A aparut o eroare la logarea in aplicatie. Va rugam sa verificati ca ati introdus corect numarul de telefon si codul de acces, iar daca eroarea persista va rugam contactati serviciul de suport la numarul 0767198682.' }`

Reply:
Yes indeed. Let's keep this as a quick fix and log a new one for having the error string as part of the configuration.

Reply:
My bad, we already have it :)
Niiiiiiice!